<launch>
    <node pkg="rosserial_python" type="serial_node.py" name="serial_node_com7" output="screen" respawn="true" >
        <param name="port" value="COM8" type="string" />
        <param name="baud" value="500000" type="int" />
        <remap from="led8_data8" to="zzz/led8"/>
        <!--remap from="zr_hw_cmd" to="zzzros/srv_cmd"/-->
    </node>
    <node pkg="zr_ledkey_1638" type="ledkey_pc.py" name="ledkey_pc_node" output="screen" respawn="true" >
        <remap from="led8_data8" to="zzz/led8"/>
        <param name="rate_param" value="1" type="int" />
    </node>

    <!--node pkg="zr_hw_cmd" type="cmd_srv_client.py" name="cmdclient" output="screen" respawn="true" >
        
        <param name="srv_param" value="zzzros/srv" type="string" />
        <remap from="zr_hw_cmd" to="zzzros/srv_cmd"/>
    </node-->
</launch>